custom_components/toyota/__init__.py

```diff
@@ -88,7 +91,7 @@ async def async_update_data():
             for car in cars:
                 vehicle = await client.get_vehicle_status(car)
 
-                if vehicle.is_connected:
+                if vehicle.is_connected:  # Clarify the 'Vehicle' members

```

Comment:
Hi @DurgNomis-drol,
can you please explain the dependencies for `vehicle.is_connected`, `vehicle.odometer` and `vehicle.statistics` a little bit further for me?
It seems that none of them is a member of the `Vehicle` class returned by the `get_vehicle_status()` function.

Reply:
Has been clarified here:
https://github.com/DurgNomis-drol/mytoyota/pull/221#issuecomment-1456105150
The cause is breaking changes between mytoyota 0.7.8 and 0.9.1. The mytoyota version here has therefore been pinned to 0.7.8 for the time being.